mythical-creatures/spec/medusa_spec.rb

```diff
@@ -25,17 +25,40 @@
   it 'turns a person to stone when staring at them' do
     medusa = Medusa.new('Cassiopeia')
     victim = Person.new('Perseus')
-
-    assert(victim.stoned?).to be false
+    
+    expect(victim.stoned?).to be false
     medusa.stare(victim)
-    assert(victim.stoned?).to be true
+    expect(victim.stoned?).to be true
   end
-
+  
   it 'can only have three victims' do
     # your code here
-  end
+    medusa = Medusa.new('Cassiopeia')
+    victim = Person.new('Perseus')
+    victim1 = Person.new('Perseus1')
+    victim2 = Person.new('Perseus2')
 
+    medusa.stare(victim)
+    medusa.stare(victim1)
+    medusa.stare(victim2)
+    
+    expect(medusa.capped?).to be true
+  end
+  
+  
   it 'if a fourth victim is stoned the first is unstoned' do
     # your code here
+    medusa = Medusa.new('Cassiopeia')
+    victim = Person.new('Perseus')
+    victim1 = Person.new('Perseus1')
+    victim2 = Person.new('Perseus2')
+    victim3 = Person.new('Perseus3')
+
+    medusa.stare(victim)
+    medusa.stare(victim1)
+    medusa.stare(victim2)
+    medusa.stare(victim3)
```

Comment:
I like the structure of this test so far. Another way to write this test could be:

```
expect(victim.stoned?).to be false
medusa.stare(victim)
expect(victim.stoned?).to be true

expect(victim1.stoned?).to be false
medusa.stare(victim1)
expect(victim1.stoned?).to be true

expect(victim2stoned?).to be false
medusa.stare(victim2)
expect(victim2.stoned?).to be true

expect(victim3.stoned?).to be false
medusa.stare(victim3)
expect(victim3.stoned?).to be true
expect(victim.stoned?).to be false
```

This type of test increases our confidence that the object's state is what we expected, and therefore when an action is performed, it _did_ indeed change that state to be something else.